mobile/README.md

```diff
@@ -114,11 +159,90 @@ to your project to support the generated code.
 
 ### Android
 
+####First option:
+
 In order to generate protobuf definitions for Android, add `--java_out=.`
 
 to the first `protoc` invocation found in
 [ `gen_protos.sh` ](../lnrpc/gen_protos.sh). Then, run `make rpc`.
 
+
+####Second option (preferable):
```

Comment:
IMO "preferable" is subjective here. Perhaps just add `(for Gradle based projects)` instead?

Reply:
The thing is that this is the only working way for Android. You could build `.java` files manually and manually import them into the project, but it won't work. At least I was not able to make it work. That is why I marked it as `preferable`. If you don't like it, sure, I will change it.

Reply:
Did something change with Android that the first way doesn't work anymore? Or did that never work? Perhaps we should just replace the first option if you say it doesn't work anyway.

Reply:
I can't tell you for sure, but until I got help with that, I tried to build and import them manually and after that, a lot of requests didn't work. It was not able to parse responses, something with java protobuf dependency.